presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve dictionary writer performance.
+* Add BigQuery Connector Plugin.
+* Added support for memozing dead nodes. This can be enabled by setting the ``internal-communication.memoize-dead-nodes-enabled`` configuration property.
+* Enforce NOT NULL constraint by checking values for NOT NULL columns.
+* Remove hive-apache dependency on presto-orc.
```

Comment:
I would remove this, it's hard to see how this is relevant to any user of Presto.